tools/build/README.md

```diff
@@ -17,6 +17,7 @@ The script will skip build steps whose inputs have not changed since the last ru
 
 - On Windows, `BUILD.bat` will automatically install a private (vendored) copy of Node.
 - On Linux, install Node using your package manager or from <https://nodejs.org/en/download/>.
+- On Linux you will likely need to compile rust-g, for instructions see https://github.com/tgstation/rust-g
```

Comment:
Not sure if a comma is needed here as in: `On Linux, you....`

Reply:
Replace the comma you already have with a period, then put a comma after linux. Comma splicing kills!

Reply:
```suggestion
- On Linux you will need to compile rust-g, for instructions see https://github.com/tgstation/rust-g
```
its not packaged so it's garaunteed

Reply:
tgs has scripts for adding it I think?  @bobbahbrown @Cyberboss

Reply:
mentioned in the hosting documentation

Reply:
PreCompile.sh for /tg/ and consequently tgmc has it

Reply:
@Cyprex change it to say that unless using tgs4 or later that you will need to build/obtain your own rust_g.so

Reply:
No need to specify 4 or later, cause 1-3 are windows only :honk: